src/util/array2string.cpp

```diff
@@ -0,0 +1,21 @@
+#include <util/array2string.h>
+
+void array2string(const symbolt &src, exprt &dest)
+{
+  if(src.type.id() != irept::id_array)
+    return;
+
+  std::string value_str;
+  size_t cnt = 0;
+  forall_operands(it, src.value)
+  {
+    std::string op_str = it->cformat().as_string();
+    if(cnt < src.value.operands().size() - 1)
+      value_str.push_back(op_str[1]);
```

Comment:
Would it be possible to add a comment here explaining why "op_str[1]"?

Reply:
Sure. I will add some comments. Basically it's the reversal of the `string2array()`:
```c++
```